public/index.html

```diff
@@ -31,6 +32,29 @@ <h1>Guess My Number!</h1>
           </p>
       </section>
   </main>
+
+  <!-- Modal -->
+  <div class="modal hidden">
+    <button class="btn btn__close-modal close-modal">&times;</button>
+    <h2>How to play</h2>
+    <div class="text-container">
+      <div>
+        <p>
+          It's easy! Guess a number between 1 and 20, and type it into the yellow rectangle, then click "Check".
```

Comment:
```suggestion
          It's easy! Guess a number between 1 and 20, and type it into the yellow rectangle, then click "Check".
```
The highest number (20) should not be hard-coded.